kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue

```diff
@@ -164,6 +179,9 @@
       actions: {
         createUser: actions.createUser,
       },
+      getters: {
+        facility_id: state => state.pageState.facility_id,
```

Comment:
Adding a getter for pageState because actions shouldn't be able to access pageState.